ersilia/core/model.py

```diff
@@ -49,7 +49,7 @@ def __init__(
         fetch_if_not_available=True,
         preferred_port=None,
         log_runs=True,
-        track_runs=False,
+        track_status=False,
```

Comment:
I think we can either keep the variable name as `track` or `track_runs` as it previously was.